app/javascript/components/Pages/Home/index.jsx

```diff
@@ -39,7 +39,9 @@ const Home = ({
   timeTitle,
   familyTitle,
   skillsTitle,
-  homeTitle
+  homeTitle,
+  logo,
+  alt_logo
```

Comment:
This is just me being picky 😄 but can we try to stick to one variable naming convention? We're mostly using camelCase but I can see there's some snake_case that's snuck in here

Reply:
I always need some help for names, so please, be picky 😆 
It's a bit confuse, few places we have Camel and others no, I changed :D